### PR TITLE
chore: extend "config:js-lib"

### PR DESCRIPTION
**Pull request by RebeccaStevens**
## PR Checklist

- [ ] Addresses an existing open issue
- [ ] That issue was marked as [`status: accepting prs`](https://github.com/JoshuaKGoldberg/ts-api-tools/issues?q=is%3Aopen+is%3Aissue+label%3A%22status%3A+accepting+prs%22)
- [x] Steps in [CONTRIBUTING.md](https://github.com/JoshuaKGoldberg/ts-api-tools/blob/main/.github/CONTRIBUTING.md) were taken

## Overview

This is a JS lib so we should probably use the recommened renovate config for such.

### CODE REVIEW

File: .github/renovate.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
 	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
+	"extends": ["config:js-lib"],
```
Comment (JoshuaKGoldberg) [inline .github/renovate.json:3]: I poked through the docs at https://docs.renovatebot.com/presets-config/#configjs-lib a bit but didn't see much info. Is there a reason to do this? I've been pretty happy with the config so far.

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
hmm, I'm surprised this isn't failing a lint check on sorted package sorts.

I'm neutral on the change so far 😄 but if there's a compelling case to add it, I'm down - here and in [template-typescript-node-package](https://github.com/JoshuaKGoldberg/template-typescript-node-package)!

File: .github/renovate.json (hunk 1 above)
Comment (RebeccaStevens) [reply .github/renovate.json:3]: These are the configs I'd personally recommend extending from:

```json
{
  "extends": [
    ":ignoreModulesAndTests",
    ":preserveSemverRanges",
    ":pinDevDependencies",
    ":widenPeerDependencies",
    "group:monorepos",
    "group:recommended",
    "workarounds:typesNodeVersioning"
  ]
}
```

Comment (RebeccaStevens) [reply .github/renovate.json:3]: This is roughly the config I use if you are curious.

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    ":ignoreModulesAndTests",
    "group:monorepos",
    "group:recommended",
    ":automergeAll",
    ":automergePr",
    ":semanticCommits",
    ":prHourlyLimitNone",
    ":prConcurrentLimitNone",
    "helpers:disableTypesNodeMajor",
    "workarounds:typesNodeVersioning"
  ],
  "labels": ["Type: Maintenance", ":blue_heart:"],
  "automergeStrategy": "rebase",
  "packageRules": [
    {
      "matchManagers": ["npm"],
      "matchDepTypes": ["dependencies"],
      "rangeStrategy": "update-lockfile",
      "semanticCommitType": "chore",
      "semanticCommitScope": "deps",
      "major": {
        "semanticCommitType": "build"
      }
    },
    {
      "matchManagers": ["npm"],
      "matchDepTypes": ["devDependencies"],
      "rangeStrategy": "pin",
      "semanticCommitType": "chore",
      "semanticCommitScope": "dev-deps"
    },
    {
      "matchManagers": ["npm"],
      "matchDepTypes": ["peerDependencies"],
      "rangeStrategy": "widen",
      "semanticCommitType": "chore",
      "semanticCommitScope": "peer-deps",
      "major": {
        "semanticCommitType": "build"
      }
    },
    {
      "matchManagers": ["github-actions"],
      "rangeStrategy": "replace",
      "semanticCommitType": "ci",
      "semanticCommitScope": "dev-deps"
    }
  ]
}
```

**Comment by RebeccaStevens** [comment]
This change is kind of insignificant so feel free to close it.

**Comment by JoshuaKGoldberg** [comment]
👍 I haven't had any issues from the config yet, so closing this for now. Thanks for sending though!

**Pull request closed without merging**